overrides/ios-override.json

```diff
@@ -4,6 +4,23 @@
             "exceptions": [
             ]
         },
+        "incrementalRolloutTest2": {
```

Comment:
I chatted with @samsymons before implementing this and we chose `incrementalRolloutTest2` at his suggestion to ensure there could be no confusion with the macOS test.

This is has been already been implemented in [BSK](https://github.com/duckduckgo/BrowserServicesKit/pull/502/files) and with the iOS release due to start rollout on Monday it's too late to change the feature name unfortunately.

Reply:
Fair enough. Let's consider it further for future cases.

Reply:
Thank you & will do 👍

Reply:
Yeah that one's on me - I had wanted to avoid us re-enabling this for macOS, forgetting that this would have only applied to the iOS config anyway and macOS wouldn't have been affected.